paysage/models/hidden.py

```diff
@@ -34,17 +37,56 @@ def __init__(self, layer_list):
         # the layers are stored in a list with the visible units
         # as the zeroth element
         self.layers = layer_list
+        self.num_layers = len(self.layers)
 
-        assert len(self.layers) == 2,\
+        assert self.num_layers == 2,\
         "Only models with 2 layers are currently supported"
 
         # adjacent layers are connected by weights
         # therefore, if there are len(layers) = n then len(weights) = n - 1
         self.weights = [
         layers.Weights((self.layers[i].len, self.layers[i+1].len))
-        for i in range(len(self.layers) - 1)
+        for i in range(self.num_layers - 1)
         ]
 
+    def get_config(self):
+        """
+        Get a configuration for the model.
+
+        Notes:
+            Includes metadata on the layers.
+
+        Args:
+            None
+
+        Returns:
+            A dictionary configuration for the model.
+
+        """
+        config = {
+        "model type": "RBM",
+        "layers": [ly.get_config() for ly in self.layers],
+        "layer_types": ["visible", "hidden"],
```

Comment:
This is a placeholder for now for layer names, which we'll need for deep models. It probably should go into the layer as an attribute, though, now or at that point.